packages/material-ui/src/TablePagination/TablePagination.test.js

```diff
@@ -200,29 +201,28 @@ describe('<TablePagination />', () => {
       assert.strictEqual(page, 0);
     });
 
-    it('should display 0 as start number if the table is empty ', () => {
+    it('should handle when count is out of range', () => {
+      let page = 1;
       const wrapper = mount(
         <table>
-          <TableFooter>
+          <TableBody>
             <TableRow>
               <TablePagination
-                count={0}
-                page={0}
+                count={-1}
+                page={page}
+                onChangePage={(event, nextPage) => {
+                  page = nextPage;
+                }}
                 rowsPerPage={10}
-                onChangePage={noop}
-                onChangeRowsPerPage={noop}
               />
             </TableRow>
-          </TableFooter>
+          </TableBody>
         </table>,
       );
-      assert.strictEqual(
```

Comment:
What do you think of asserting the label for page 1 and 2? :)

Reply:
What do you mean? The range should be more? :)

Reply:
I was thinking of an assertion close to

page 1: "1-10 of more than 10"
page 2:  "11-20 of more than 20"

Reply:
just leave it or adding changes?

Reply:
My concern is that it's not obvious that the rendered text will keep working as expected with `count={-1}`. How do we make sure it does?

Reply:
I took the example of a guy who made a bug report as a basis. The behaviour will be the same as in the sandbox example. I want to delve deeper into this issue, and I want to hear your opinion on this subject in more details :)